src/Components/VideoCard/video-card.js

```diff
@@ -0,0 +1,31 @@
+import '../VideoCard/video-card.css';
+import { useNavigate } from 'react-router-dom';
+const VideoCard = ({ video }) => {
+  const navigate = useNavigate();
+    const {_id,title,catchName,uploaded} = video;
```

Comment:
Suggestion: use some code formatters, that will make your code clean by providing necessary gaps and spaces